1-js/11-async/07-microtask-queue/article.md

```diff
@@ -93,11 +93,11 @@ setTimeout(() => promise.catch(err => alert('caught')), 1000);
 window.addEventListener('unhandledrejection', event => alert(event.reason));
 ```
 
-Now if we run it, we'll see `Promise Failed!` first and then `caught`.
+Now, if we run it, we'll see `Promise Failed!` first and then `caught`.
 
 If we didn't know about the microtasks queue, we could wonder: "Why did `unhandledrejection` handler run? We did catch and handle the error!"
 
-But now we understand that `unhandledrejection` is generated once the microtask queue is complete. The engine examines promises and, if any of them is in the "rejected" state, then the event triggers.
+But now we understand that `unhandledrejection` is generated when the microtask queue is complete: The engine examines promises and, if any of them is in the "rejected" state, then the event triggers.
 
```

Comment:
Style guides may differ on this, but I strongly prefer to not capitalize after the colon here: "...is complete; the engine examines...".